zds/forum/forms.py

```diff
@@ -20,6 +21,13 @@ class TopicForm(forms.Form):
             attrs={
                 'placeholder': _(u'Titre de mon sujet'),
                 'required': 'required',
+                'data-autocomplete': json.dumps({
+                    'type': 'simple',
+                    'fieldname': 'title',
+                    'header': str(_(u'Sujets similaires :')),
```

Comment:
@Situphen la fonction d'i18n, là, qu'on utilise d'ailleurs partout, est lazy. Certaines autres fonctions ne forcent pas l'évaluation de ce `_()` lazy, et ça a tendance à lever une exception parfois surprenante. Ici, c'est très probablement ce que @AmarOk1412 a rencontré. `str()` forçant l'évaluation du machin, ça passe avec.